### DISCUSSION TITLE
Running on a computer with low memory

### DISCUSSION THREAD

**Discussion by vrennie** [TORCH-Consortium/MAGMA#183] [Q&A]
Hi @abhi18av,

It appears that changing the conda_local config (see pull request) does not result in a true cap on the mem and cpus?

Could you clarify how to run on a low memory laptop?

**Comment by abhi18av** [comment]
(NOTE: Moved from an issue to a discussion)

Sure Vincent - I have added an explanation on the PR itself :) 

https://github.com/TORCH-Consortium/MAGMA/pull/181#discussion_r1395635371
